guides/common/modules/proc_adding-a-proxmox-connection-to-server.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Adding_a_Proxmox_Connection_to_{project-context}_{context}"]
+= Adding a Proxmox Connection to {ProjectServer}
+
+Use this procedure to add a Proxmox connection to {Project}.
+
+.Prerequisites
+* Ensure that the host and network-based firewalls are configured to allow communication from {Project} to Proxmox on TCP port 443.
```

Comment:
```suggestion
* Ensure that the host and network based firewalls are configured to allow communication from {Project} to Proxmox on TCP port 443.
```

Reply:
No change; `git grep` tells me it's "network-based".